upgrade/UPGRADE-v11.0.0-dev.md

```diff
@@ -646,3 +646,52 @@ There you can find links to upgrade notes for other versions too.
         - onKernelException(GetResponseForExceptionEvent $event)
         + onKernelException(ExceptionEvent $event): void
         ```
+
+## Composer dependencies
+
+- replace swiftmailer with symfony/mailer [#2470](https://github.com/shopsys/shopsys/pull/2470)
+    - see #project-base-diff
+    - the mail spooling functionality has been removed without replacement
```

Comment:
shouldn't be mentioned that now we configure mail transport with DSN instead of separate varaibles?

Reply:
good idea, 65948a696